docs/aws/audit/cloudtrailmonitoring/rules/cloudtrail_global_services_enabled.mdx

```diff
@@ -23,6 +23,221 @@ AWSWAF, GDPR, PCIDSS
+</Accordion>
+
+<Accordion title='Using CLI'>
+To ensure that AWS CloudTrail records both regional and global events using the AWS CLI, follow these steps:
+
+1. **Create or Update a CloudTrail Trail:**
+   Ensure you have a CloudTrail trail that is configured to log both management and data events. If you don't have a trail, create one. If you already have a trail, you can update it.
+
+   ```sh
+   aws cloudtrail create-trail --name my-trail --s3-bucket-name my-bucket
+   ```
+
+2. **Enable Logging for Global Services:**
+   Update the trail to include global service events (e.g., IAM, STS, CloudFront).
+
+   ```sh
+   aws cloudtrail update-trail --name my-trail --include-global-service-events
+   ```
+
+3. **Enable Multi-Region Trail:**
+   Ensure the trail is configured to log events from all regions.
+
+   ```sh
+   aws cloudtrail update-trail --name my-trail --is-multi-region-trail
+   ```
+
+4. **Start Logging:**
+   Start logging for the trail to ensure that events are being recorded.
+
+   ```sh
+   aws cloudtrail start-logging --name my-trail
+   ```
+
+By following these steps, you can ensure that your CloudTrail is configured to record both regional and global events.
+</Accordion>
+
+<Accordion title='Using Python'>
+To ensure that AWS CloudTrail records both regional and global events using Python scripts, you can use the `boto3` library, which is the AWS SDK for Python. Here are the steps to achieve this:
+
+1. **Install Boto3 Library**:
+   Ensure you have the `boto3` library installed. If not, you can install it using pip:
+   ```bash
+   pip install boto3
+   ```
+
+2. **Initialize Boto3 Client**:
+   Initialize the `boto3` client for CloudTrail.
+
+3. **Create or Update CloudTrail**:
+   Create a new CloudTrail or update an existing one to ensure it records both regional and global events.
+
+4. **Verify Configuration**:
+   Verify that the CloudTrail configuration includes both regional and global events.
+
+Here is a Python script to achieve this:
+
+```python
+import boto3
+
+# Initialize a session using Amazon CloudTrail
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'  # Specify your region
+)
+
+# Initialize CloudTrail client
+cloudtrail_client = session.client('cloudtrail')
+
+# Define the trail name
+trail_name = 'your-trail-name'
+
+# Create or update the CloudTrail to record both regional and global events
+try:
+    response = cloudtrail_client.update_trail(
+        Name=trail_name,
+        IncludeGlobalServiceEvents=True
+    )
+    print(f"Successfully updated trail {trail_name} to include global service events.")
+except cloudtrail_client.exceptions.TrailNotFoundException:
+    response = cloudtrail_client.create_trail(
+        Name=trail_name,
+        S3BucketName='your-s3-bucket-name',  # Specify your S3 bucket name
+        IncludeGlobalServiceEvents=True,
+        IsMultiRegionTrail=True
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>